app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,93 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>As a computing teacher you'll help pupils become responsible and confident users of information and technology in an increasingly digital world.</p>
+<p>Whether it's teaching your pupils about artificial intelligence (AI) or creating apps, you could inspire them to look at a future role in technology.</p>
+<p>Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee computing teachers.</p>
+</section>
+  <section class="col col-space-s col-content-max">
+<%= render TextBlockWithTwoImagesComponent.new(
+heading: "Help your pupils learn new skills",
+colour: "pink",
+image_1_path: "static/images/content/hero-images/0035.jpg",
+image_1_alt: "",
+image_2_path: "static/images/content/hero-images/0037.jpg",
+image_2_alt: "",
+) do %>
+<p>Computing focuses on 3 areas:</p>
+<ul>
+<li>information and communication technology</li>
+<li>digital literacy</li>
+<li>computer science</li>
+</ul>
+<p>You'll inspire pupils to access new fields and teach them new, transferable skills that will open lots of sectors, from engineering to healthcare and that could lead to careers such as software engineering and web design.</p>
+<% end %>
+</section>
+<section class="col col-720">
+<h2>What you'll be teaching</h2>
+<p>At GCSE and A level you'll teach computer science. You'll teach themes from the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum</a>, with opportunities to develop your pupils' computational thinking.  </p>
```

Comment:
```suggestion
<p>You'll teach themes from the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum for computing</a>, with opportunities to develop your pupils' computational thinking.</p>
```

Not sure the GCSE and A level bit is needed here - that's mentioned lower down for KS4 and KS5?

I think it would be worth adding in `for computing` to the link so it is really clear where it goes

Also - some additional whitespaces at the end need deleting